tests/controllers/S3LinkController.ts

```diff
@@ -0,0 +1,37 @@
+import {describe, test} from '@jest/globals';
+import {containerBuilder} from '../testContainer';
+import {Request, Response} from 'express';
+import fn = jest.fn;
+import S3LinkController from '../../src/controllers/S3LinkController';
+
+describe('S3 Link Controller', () => {
+  const container = containerBuilder();
+  const s3LinkController = container.get('s3LinkController') as S3LinkController;
+
+  test('run action Default', () => {
+    const request = {
+      body: {
+        frames: ['image22.jpg', 'image33.jpg']
```

Comment:
we should get the name of the video too here right?

Reply:
yes we should